### ISSUE TITLE
Fix `boreholes.field_from_file` function to work with 1 borehole 

### ISSUE THREAD

**Issue by j-c-cook** [MassimoCimmino/pygfunction#208]
The `boreholes.field_from_file(filename)` function does not work when there is only one borehole defined in the file.

**Comment by MassimoCimmino** [comment]
The cause may simply be that `numpy.loadtxt(fname)` returns a 1d array whe nthere is a single line in the file:
https://github.com/MassimoCimmino/pygfunction/blob/88a76436aeaaa4d5a9f41e623a8a007ba9e5c780/pygfunction/boreholes.py#L848-L849

A possible fix is to use the `ndmin` argument to ensure the returned array is two-dimensional:
```
    # Load data from file
    data = np.loadtxt(filename, ndmin=2)
```

**Comment by MassimoCimmino** [comment]
The following snippet produces an error:
```
>>> import pygfunction as gt
>>> file = np.savetxt('test.txt', np.array([[ 0., 0., 150., 4., 0.075]]), delimiter=' ', header='H\tD\tr_b\tx\ty')
>>> gt.boreholes.field_from_file('test.txt')
IndexError: invalid index to scalar variable.
```

The proposed change solves the issue.

**Comment by j-c-cook** [comment]
It appears that this was merged into master, but not the `maintenance/2.2.x` release branch. I'll cherry pick the relevant commits and open up a PR.
